foreman/data_refinery_foreman/surveyor/external_source.py

```diff
@@ -82,6 +80,9 @@ def handle_batch(self, batch: Batch, key_values: BatchKeyValue = None):
                        "is of the type DiscoveryPipeline.")
             raise InvalidProcessedFormatError(message)
 
+        batch.internal_location = os.path.join(batch.accession_code,
+                                               batch.pipeline_required)
+
```

Comment:
I'm not sure why this needed to be changed. I swear this was an issue in 3 different merge conflicts I resolved and I put it afterwards anyway. This should have already been like this, but I guess maybe I messed up one of the merge conflicts?